_sass/custom/custom.scss

```diff
@@ -41,15 +41,6 @@ $content-width: 740px !default;
 $table-border-color: $grey-lt-300;
 $toc-width: 232px !default;
```

Comment:
Not sure why but removing `$toc-width` from here threw errors; will investigate later.